models/balancer/ethereum/balancer_ethereum_balances.sql

```diff
@@ -1,56 +1,57 @@
 {{
     config(
         alias = alias('balances'),
+        tags = ['dunesql'],
         post_hook='{{ expose_spells_hide_trino(\'["ethereum"]\',
                                     "project",
                                     "balancer",
-                                    \'["metacrypto", "jacektrocinski"]\') }}'
+                                    \'["metacrypto", "jacektrocinski", "viniabussafi"]\') }}'
     ) 
 }}
 
-{% set balancer_contract = "0xba12222222228d8ba445958a75a0704d566bf2c8" %}
+{% set balancer_contract = '0xba12222222228d8ba445958a75a0704d566bf2c8' %}
 
 WITH pools AS (
     SELECT pool as pools
     FROM {{ source('balancer_v1_ethereum', 'BFactory_evt_LOG_NEW_POOL') }}
 ),
 
 joins AS (
-    SELECT p.pools as pool, date_trunc('day', e.evt_block_time) AS day, e.contract_address AS token, SUM(value) AS amount
+    SELECT p.pools as pool, date_trunc('day', e.evt_block_time) AS day, e.contract_address AS token, SUM(CAST(value AS int256)) AS amount
     FROM {{ source('erc20_ethereum', 'evt_transfer') }} e
-    INNER JOIN pools p ON e.`to` = p.pools
+    INNER JOIN pools p ON e."to" = p.pools
     GROUP BY 1, 2, 3
     UNION ALL
-    SELECT e.`to` as pool, date_trunc('day', e.evt_block_time) AS day, e.contract_address AS token, SUM(value) AS amount
+    SELECT e."to" as pool, date_trunc('day', e.evt_block_time) AS day, e.contract_address AS token, SUM(CAST(value as int256)) AS amount
     FROM {{ source('erc20_ethereum', 'evt_transfer') }} e
-    WHERE e.`to` = '{{balancer_contract}}'
+    WHERE CAST(e."to" as VARCHAR)= '{{balancer_contract}}'
```

Comment:
rather than cast this, it's better to leave as varbinary and remove single quotes from the variable, then it will come through as varbianry too